toucan_connectors/google_credentials.py

```diff
@@ -37,13 +39,16 @@ class GoogleCredentials(BaseModel):
         description=f'{CREDENTIALS_INFO_MESSAGE}. You should not need to change the default value.',
     )
     client_x509_cert_url: HttpUrl = Field(
-        ...,
+        None,
         title='Client X509 certification URL',
         description=CREDENTIALS_INFO_MESSAGE,
     )
+    # ---
+    # for the jwt-token given as param
+    jwt_token: str = Field(None, title='JSON web token (JWT)', description=CREDENTIALS_INFO_MESSAGE)
 
     @validator('private_key')
-    def unescape_break_lines(cls, v):
+    def unescape_break_lines(cls, v: str = ''):
```

Comment:
should probably keep the `None` to keep all the fields to the same value if unused and use a simple condition instead